src/components/tabs/_tabs.scss

```diff
@@ -29,7 +29,7 @@
     margin-left: govuk-spacing(5);
 
     &::before {
-      content: "— ";
+      content: "\2014 ";
```

Comment:
It'd be good to add a comment just so we know what this character is without having to look it up (and also in case anyone searches for it)

```suggestion
      content: "\2014 "; // "— "
```